composer.json

```diff
@@ -15,18 +22,23 @@
 			"email": "esteban.gehring@bithost.ch"
 		}
 	],
+	"homepage": "https://github.com/bithost-gmbh/pdfviewhelpers",
 	"require": {
-		"typo3/cms-core": "^12.4 || ^13.4",
-		"php": ">=8.1.0",
+		"setasign/fpdi": "^2.0",
 		"tecnickcom/tcpdf": "^6.2",
-		"setasign/fpdi": "^2.0"
+		"typo3/cms-core": "^13.4"
```

Comment:
We would like to support both TYPO3 v12 and v13, which seems to be possible with little overhead.

Reply:
For that case we can switch to the runScript.sh of news, which has support for multiple TYPO3 versions